.github/workflows/flutter_test.yml

```diff
@@ -82,24 +83,26 @@ jobs:
         uses: reactivecircus/android-emulator-runner@d94c3fbe4fe6a29e4a5ba47c12fb47677c73656b #pin@v2.28.0
         with:
           working-directory: ./flutter/example/android
-          api-level: 21
+          api-level: 31
```

Comment:
Won't it work with api-level 21 anymore?

Reply:
We still have a test where we check build with min version. Figured the more recent emulators _might_ give us better performance and stability.